llvm/lib/Target/AIE/AIEPostPipeliner.cpp

```diff
@@ -606,21 +627,27 @@ bool PostPipeliner::tryHeuristics() {
 
   DEBUG_SUMMARY(dbgs() << "-- MinLength=" << MinLength << "\n");
 
-  static const std::tuple<int, bool, uint64_t> Configs[] = {
+  static const std::tuple<int, bool, bool, uint64_t> Configs[] = {
       // Loosely speaking, a lower value of the first parameter targets
       // a lower stage count, which benefits code size.
-      {0, false, ConfigStrategy::NodeNum},
-      {0, false, ConfigStrategy::Latest},
-      {0, true, ConfigStrategy::Critical},
-      {0, true,
+      {0, true, false, ConfigStrategy::NodeNum},
```

Comment:
Nit: `{/*ExtraStages=*/0, /*TopDown=*/true, /*Rerun=*/false, ConfigStrategy::NodeNum}`? Or at least give a top-level comment documenting the elements of the tuple?